packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -47,14 +47,16 @@ async function getRfqtIndicativeQuotesAsync(
 
 export class MarketOperationUtils {
     private readonly _wethAddress: string;
+    private readonly _multiBridge: string;
 
     constructor(
         private readonly _sampler: DexOrderSampler,
         private readonly contractAddresses: ContractAddresses,
         private readonly _orderDomain: OrderDomain,
-        private readonly _liquidityProviderRegistry: string = NULL_ADDRESS,
+        private readonly _liquidityProviderRegistry?: string,
```

Comment:
This change does seem a little inconsistent with the following multiBridge. What are your thoughts on this change?